lms/djangoapps/certificates/tests/test_cert_management.py

```diff
@@ -11,7 +12,7 @@
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory, check_mongo_calls
 from student.tests.factories import UserFactory, CourseEnrollmentFactory
-from certificates.management.commands import resubmit_error_certificates, regenerate_user, ungenerated_certs
+from certificates.management.commands import resubmit_error_certificates, regenerate_user, ungenerated_certs, rescind_cert  # pylint: disable=line-too-long
```

Comment:
there's got to be a way to import multiple functions from a file without having to disable a pylint thing...